docs/plugin-development/index.md

```diff
@@ -684,7 +703,9 @@ If `"no-treeshake"` is returned, treeshaking will be turned off for this module
 
 If `null` is returned or the flag is omitted, then `moduleSideEffects` will be determined by the `load` hook that loaded this module, the first `resolveId` hook that resolved this module, the [`treeshake.moduleSideEffects`](../configuration-options/index.md#treeshake-modulesideeffects) option, or eventually default to `true`.
 
-`attributes` contain the import attributes that were used when this module was imported. At the moment, they do not influence rendering for bundled modules but rather serve documentation purposes. If `null` is returned or the flag is omitted, then `attributes` will be determined by the `load` hook that loaded this module, the first `resolveId` hook that resolved this module, or the attributes present in the first import of this module.
+`options.attributes` contain the import attributes that were used when this module was imported, which is determined by the `load` hook that loaded this module, the first `resolveId` hook that resolved this module, or the attributes present in the first import of this module.
+
+The mean of `attributes` is same as `options.attributes`. At the moment, they do not influence rendering for bundled modules but rather serve documentation purposes. If `null` is returned or the flag is omitted, then `attributes` will be equal to `options.attributes`.
```

Comment:
Similar to `load`, I would suggest to remove this section and the mention of `attributes` in the `SourceDescription` above.